tests/integration/test_events.py

```diff
@@ -230,3 +237,66 @@ def test_put_events_with_target_lambda(self):
         self.events_client.delete_event_bus(
             Name=TEST_EVENT_BUS_NAME
         )
+
+    def test_schedule_expression_event_with_http_endpoint(self):
+        class HttpEndpointListener(ProxyListener):
+            def forward_request(self, method, path, data, headers):
+                event = json.loads(to_str(data))
+                events.append(event)
+                return 200
+
+        topic_name = 'topic-{}'.format(short_uid())
+        rule_name = 'rule-{}'.format(short_uid())
+        target_id = 'target-{}'.format(short_uid())
+        events = []
+
+        local_port = get_free_tcp_port()
+        proxy = start_proxy(local_port, backend_url=None, update_listener=HttpEndpointListener())
+        wait_for_port_open(local_port)
+
+        endpoint = '{}://{}:{}'.format(get_service_protocol(), config.LOCALSTACK_HOSTNAME, local_port)
+
+        topic_arn = self.sns_client.create_topic(Name=topic_name)['TopicArn']
+        self.sns_client.subscribe(TopicArn=topic_arn, Protocol='http', Endpoint=endpoint)
+
+        event = {
+            'env': 'testing'
+        }
+
+        self.events_client.put_rule(
+            Name=rule_name,
+            ScheduleExpression='rate(1 minutes)'
+        )
+
+        self.events_client.put_targets(
+            Rule=rule_name,
+            Targets=[
+                {
+                    'Id': target_id,
+                    'Arn': topic_arn,
+                    'Input': json.dumps(event)
+                }
+            ]
+        )
+
+        def received():
+            self.assertGreaterEqual(len(events), 2)
```

Comment:
Out of curiosity - why are we expecting 2 events here? Shouldn't it be 1?

Reply:
This point has been clarified - 1 for subscription confirmation message, 1 for event triggered by scheduled expression